controller/cmd/identity/main.go

```diff
@@ -42,6 +40,13 @@ func Main(args []string) {
 	addr := cmd.String("addr", ":8080", "address to serve on")
 	adminAddr := cmd.String("admin-addr", ":9990", "address of HTTP admin server")
 	kubeConfigPath := cmd.String("kubeconfig", "", "path to kube config")
+	controllerNS := cmd.String("controller-namespace", "", "namespace of the linkerd control plane")
+	identityScheme := cmd.String("identity-scheme", "", "scheme of the identity")
```

Comment:
this description is not very helpful.  can we have a flag help that is more informative?